x-pack/plugins/cases/public/components/configure_cases/connectors_dropdown.tsx

```diff
@@ -102,7 +114,11 @@ const ConnectorsDropdownComponent: React.FC<Props> = ({
                       margin-right: ${euiTheme.size.m};
                       margin-bottom: 0 !important;
                     `}
-                    type={getConnectorIcon(triggersActionsUi, connector.actionTypeId)}
+                    type={
```

Comment:
This is needed to avoid the following bug:

https://github.com/elastic/kibana/assets/7871006/d52fa758-2649-4489-a7d2-f8d9a91192ed

The icon class can be a lazy-loaded component. React will fall back to the nearest `Suspense` causing the flyout on each render of the SVG icon to appear and disappear. By adding suspense here we isolate the loading inside the select option. With the fix:


https://github.com/elastic/kibana/assets/7871006/ba14c06a-c7a3-49d6-a5ac-eba304d44c3f